### ISSUE TITLE
Support for shifted zero point or uint8 after Relu activations

### ISSUE THREAD

**Issue by SimonKeisala** [NVIDIA/TensorRT#2081]
## Description

According to a presentation material from 2017 ([8-bit Inference with TensorRT](https://on-demand.gputechconf.com/gtc/2017/presentation/s7310-8-bit-inference-with-tensorrt.pdf)) some open challenges / improvements are mentioned, whereas one being to support unsigned int8 after ReLU activations. This support still does not seem to exist, and practically result in 7-bit quantization for ReLU activations.
Are there any plans / ongoing work to add uint8 quantization, or the possibility to have either -128 and 0 as zero point in the model?

## Environment

**TensorRT Version**: 8.0.3
**NVIDIA GPU**: N/A
**NVIDIA Driver Version**: 470.103.01
**CUDA Version**: 11.4
**CUDNN Version**: N/A
**Operating System**: N/A
**Python Version (if applicable)**: N/A 
**Tensorflow Version (if applicable)**: N/A
**PyTorch Version (if applicable)**: N/A
**Baremetal or Container (if so, version)**: N/A

## Relevant Files

N/A

## Steps To Reproduce

Running trtexec --int8 on any model with QDQ having non-zero zero shift.

**Comment by nvpohanh** [comment]
This is currently not on our roadmap. Thanks

**Comment by nvpohanh** [comment]
Is there any example where 8-bit quantization works much better than 7-bit quantization? In our experience, 7-bit quantization is sufficient for most cases.

**Comment by SimonKeisala** [comment]
I will investigate the impact of using 7-bit quantization more thoroughly. The current experiments have involved using 8-bit unsigned quantization during QAT, and as a post-processing step rescale the range to signed. This resulted in a large degradation to some object detection functions.
To make a fair comparison I'll check how it performs when the QAT is done with signed symmetric quantization throughout the network for both weights and activations.

**Comment by SimonKeisala** [comment]
The latest investigation indicate that 7-bit quantization should be sufficient, with no notable performance loss when using fake-quantized inferenc. However, going from fake-quantized inference to TensorRT 8-bit quantized inference still result in a large performance drop.
The problem thus seem to be in the conversion from a QDQ-structured onnx-model to a tensorrt engine, which is a different issue compared to the original question/issue, so this issue will be closed and we'll have to do further investigations to figure out where the discrepancy between fake-quantized and tensorrt hardware quantized inference originate from.
